sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs

```diff
@@ -2898,12 +2898,18 @@ public partial class MongoDBPrivilege
     {
         public MongoDBPrivilege() { }
         public System.Collections.Generic.IList<string> Actions { get { throw null; } }
-        public Azure.ResourceManager.CosmosDB.Models.PrivilegeResource Resource { get { throw null; } set { } }
+        public Azure.ResourceManager.CosmosDB.Models.MongoDBPrivilegeResourceInfo Resource { get { throw null; } set { } }
+    }
+    public partial class MongoDBPrivilegeResourceInfo
+    {
+        public MongoDBPrivilegeResourceInfo() { }
+        public string Collection { get { throw null; } set { } }
+        public string DB { get { throw null; } set { } }
```

Comment:
This property name is too generic, can we rename it?

Reply:
Updated to `DBName`